library/misc/datafixerupper/src/main/java/org/quiltmc/qsl/datafixerupper/api/SimpleFixes.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.datafixerupper.api;
+
+import java.util.Map;
+import java.util.Objects;
+
+import com.google.common.collect.ImmutableMap;
+import com.mojang.datafixers.DataFix;
+import com.mojang.datafixers.DataFixerBuilder;
+import com.mojang.datafixers.schemas.Schema;
+
+import net.minecraft.datafixer.fix.BiomeRenameFix;
+import net.minecraft.datafixer.fix.BlockNameFix;
+import net.minecraft.datafixer.fix.ItemNameFix;
+import net.minecraft.datafixer.schema.IdentifierNormalizingSchema;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Provides methods to add {@link DataFix}es to {@link DataFixerBuilder}s.
+ */
+public final class SimpleFixes {
+	private SimpleFixes() { }
+
+	/**
+	 * Adds a block rename fix to the builder.
```

Comment:
Definitely requires an explanation of what the identifiers are meant to be

Are they supposed to respect the normal identifiers format?
Why a string?

Reply:
Data fixers use raw strings, since they're not tied to Minecraft (DFU is a separate library).

AFAIK, these strings are supposed to respect the identifier format. I will add this to the javadoc.

Reply:
Actually, I have a better idea....